hoot-services/src/main/resources/db/db.properties.in

```diff
@@ -41,6 +41,9 @@ PUBLIC_OSMAPI_VERSION = ${OSM_API_VERSION}
 # URL for the public OSM Overpass api
 PUBLIC_OVERPASS_URL = ${OVERPASS_API_URL}
 
+# URL for the public OSM Overpass api
+PRIVATE_OVERPASS_URL =
+
```

Comment:
Let's add the ability to configure a user cert for this connection to be used in dev env.